File: td3_basis.R
library(tidyverse)
titanic <- read.csv('titanic_data.csv')
data("anscombe")
data("UCBAdmissions")

titanic$Pclass <- factor(titanic$Pclass)
titanic$SurvivedFact <- factor(titanic$Survived)
titanic <- titanic %>% filter(!is.na(titanic$Age))

File: td3_basis.py
import pandas as pd 
import matplotlib.pyplot as plt 

titanic = pd.read_csv('titanic_data.csv')
anscombe = pd.read_csv('anscombe.csv')
admissions = pd.read_csv('UCBAdmissions.csv')

titanic = titanic[~titanic.Age.isna()]